index.html

```diff
@@ -80,7 +85,7 @@ <h3>Tonic</h3>
         </div>
       </div>
       <div class="card">
-        <img src="assets/images/mobile/portfolio3.svg" alt="project1">
+        <div class="project3"></div>
         <div class="card-text">
           <div class="card-header">
             <h3>Tonic</h3>
```

Comment:
Projects do not follow the design on laptop view. KIndly see the screenshot below and fix the issues.
![project1](https://user-images.githubusercontent.com/35504149/135630838-1d62cf71-9a0b-4875-8df8-e1d091182915.png)